### DISCUSSION TITLE
Flag displayed on the front page ?

### DISCUSSION THREAD

**Discussion by Warmono** [nlemoine/acf-country#42] [Q&A] [answered]
Hi,

I try to display flags (multiple countries) on the front page, i'm using Wpbakery (page builder) with element Advanced custom field, i get only the country code. 

Can u support my request ?

**Comment by nlemoine** [comment, marked as answer]
Hello, 

Sorry, this is a front-end issue. This plugin only outputs data. It's your responsability to display the corresponding flag.
